spacemacs/packages.el

```diff
@@ -1391,6 +1391,12 @@ If ARG is non nil then `ag' and `pt' and ignored."
                  dotspacemacs-search-tools)))
           (call-interactively (spacemacs//helm-do-search-find-tool tools))))
 
+      ;; evilify the helm-grep buffer
+      (evilify helm-grep-mode helm-grep-mode-map
+               (kbd "RET") 'helm-grep-mode-jump
```

Comment:
When I use `RET` is closes the buffer. I think we should bind `RET` to other-window.